mcs/class/System/System.Net.NetworkInformation/IPInterfaceProperties.cs

```diff
@@ -54,29 +54,33 @@ public override IPv6InterfaceProperties GetIPv6Properties ()
 		}
 #if MONODROID
 		[DllImport ("__Internal")]
-		static extern int _monodroid_get_dns_servers (out IntPtr dns_servers_array);
+		static extern int _monodroid_get_dns_servers (ref IntPtr dns_servers_array);
```

Comment:
Why the change from `out` to `ref`?

Reply:
Because we pass the variable already initialized to `IntPtr.Zero`.